docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+4+| Create operations
 
 |`* * *` |user |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |add tags to a person |categorise the people I know
+
+|`* * *` |user |add relationships between people |show that two persons know each other
+
+|`* *` |user |add link:#named-relationship[named relationships] between people |show that two persons know each other via a specific method
```

Comment:
It's more about the type of relationship, e.g. cousin, 2 years older, argued one year ago, now in difficult relationship, but less about how you know the person

Reply:
I believe this can be a separate use case - is that okay?

Reply:
but your definition of named-relationship is : the status between two people. I dont think "how i got to know u" is a status...if u speaking of putting "how i got to know u" as a separate use case, i think it will be fine (but not classified under named relationship). But I dont think "how i got to know u" is very important. so probably put * * ?

Reply:
Status, in this case is intended to be roughly analogous to Facebook relationship statuses: a short label / category describing how two people know each other (friends/colleagues/school). The reason for the brevity of the label is so that it is possible to do searches based on these labels (eg. display all friends of a person). 

While the long form relationship information you describe is certainly valuable to have, the free-form nature means that it is harder to do structured searches on them, which is why I believe these two things are separate user stories, since they fulfill different purposes.

Reply:
oh okie, now i get what u mean by 'status'. Emmm ya it makes sense i think